ta/pkcs11/src/object.c

```diff
@@ -470,6 +493,8 @@ enum pkcs11_rc entry_find_objects_init(struct pkcs11_client *client,
 	struct obj_attrs *req_attrs = NULL;
 	struct pkcs11_object *obj = NULL;
 	struct pkcs11_find_objects *find_ctx = NULL;
+	struct handle_db *soh_db = NULL;
```

Comment:
rename `soh_db` to `object_db`? easier to read out.

Reply:
OK